docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -49,12 +49,13 @@ repositories {
 }
 
 dependencies { <2>
-    compileOnly 'io.quarkus:quarkus-resteasy:{quarkus-version}'
+    implementation 'io.quarkus:quarkus-resteasy:{quarkus-version}'
 }
 ----
 
 <1> The {project-name} plugin needs to be applied.
 <2> This dependency is needed for a REST application similar to the getting started example.
+Since {project-name} has no container we also need this dependency for running tests, to provide this we use the `implementation` configuration.
```

Comment:
`container` is misleading here. Maybe you can simplify as:
The second dependency is used for running our tests. The test framework starts the servers automatically.

Reply:
Hm, it's both for tests and compile, it's the closest thing we get to `provided` in Gradle. My thought was to explain why we use the `implementation` configuration. The main reason is that we need it both for the compile and test. I see that the wording isn't great....
How about: " Quarkus also need this dependency for running tests etc..." ?

Reply:
Yes, like this.